techniques/aria/ARIA11.html

```diff
@@ -134,13 +137,13 @@ <h2>Related Techniques</h2>
    <section id="resources"><h2>Resources</h2>   
       <ul>
          <li>
-            <a href="https://www.w3.org/TR/wai-aria-practices/">WAI-ARIA Authoring Practices</a> 
+            <a href="https://www.w3.org/TR/wai-aria-practices-1.1/#aria_landmark">WAI-ARIA Authoring Practices 1.1, Landmark Regions</a> 
          </li>
          <li>
-            <a href="https://www.w3.org/TR/wai-aria/#usage_intro">Accessible Rich Internet Applications (WAI-ARIA), Using WAI-ARIA Roles</a>
+            <a href="https://www.w3.org/WAI/tutorials/page-structure/regions/">W3C Web Accessibility Tutorials, Page Regions</a>
          </li>
          <li>
-            <a href="https://www.w3.org/TR/wai-aria/#states_and_properties">Accessible Rich Internet Applications (WAI-ARIA), Supported States and Properties</a>
+            <a href="https://www.w3.org/TR/wai-aria-1.1/#landmark_roles">Accessible Rich Internet Applications (WAI-ARIA) 1.0, Landmark Roles</a>
```

Comment:
link to the latest version of the spec. also remove the 1.0 from the link text, since that wasn't correct / unneeded if we just link to the latest version of the spec.

```suggestion
            <a href="https://www.w3.org/TR/wai-aria/#landmark_roles">Accessible Rich Internet Applications (WAI-ARIA), Landmark Roles</a>
```